packages/material-ui/test/integration/Menu.test.js

```diff
@@ -120,15 +135,12 @@ describe('<Menu /> integration', () => {
   });
 
   it('focuses the selected item when opening', () => {
-    const { getAllByRole, getByLabelText } = render(
-      <SimpleMenu transitionDuration={0} selectedIndex={2} />,
-    );
-    const button = getByLabelText('open menu');
-
-    expect(document.body).to.be.focused;
+    const { getAllByRole, getByLabelText } = render(<ButtonMenu selectedIndex={2} />);
 
+    const button = getByLabelText('open menu');
     button.focus();
-    fireEvent.click(button);
+    button.click();
```

Comment:
This is saver because jsodm only checks `disabled` on `click` not when dispatching a click event. See https://github.com/jsdom/jsdom/issues/2665